cmake/sycl/Platform/Linux-AdaptiveCpp-SYCL.cmake

```diff
@@ -0,0 +1,31 @@
+# VecMem project, part of the ACTS project (R&D line)
+#
+# (c) 2022-2024 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# Use the standard GNU compiler options for hipSYCL.
+include( Platform/Linux-GNU )
+__linux_compiler_gnu( SYCL )
```

Comment:
For my education: where to these come from?

Reply:
https://gitlab.kitware.com/cmake/cmake/-/blob/master/Modules/Platform/Linux-GNU.cmake?ref_type=heads

And yes, as the function name implies, these are internal CMake functions. But we're way beyond trivial stuff with this code here...